scripts/win-7-update-sp1.ps1

```diff
@@ -13,7 +13,7 @@ Write-Host "$(Get-Date -Format G): Downloading and installing Windows 7 Service
 Write-Host "$(Get-Date -Format G): This process can take up to 30 minutes."
 
 Write-Host "$(Get-Date -Format G): Downloading Windows 7 Service Pack 1"
-(New-Object Net.WebClient).DownloadFile("https://download.microsoft.com/download/0/A/F/0AFB5316-3062-494A-AB78-7FB0D4461357/windows6.1-KB976932-X64.exe", "C:\Updates\windows6.1-KB976932-X64.exe")
+(New-Object Net.WebClient).DownloadFile("http://download.windowsupdate.com/msdownload/update/software/svpk/2011/02/windows6.1-kb976932-x64_74865ef2562006e51d7f9333b4a8d45b7a749dab.exe", "C:\Updates\windows6.1-KB976932-X64.exe")
```

Comment:
Can we use https here?

Reply:
OK. However, tried to verify if works before committing the fix and Powershell in Win7 throws a "Could not establish trust relationship for the SSL/TLS secure channel." (same in curl, Firefox etc SSL_ERROR_BAD_CERT_DOMAIN).
Looked for other official http_s_ location of KB, but couldn't find any.
Suggestion welcome how to fix/proceed.

Reply:
Alright. Oh that's a classic one I struggled way too often PowerShell 😅 
I always have to look it up, but I've added it to my PowerShell tricks page: https://github.com/StefanScherer/dockerfiles-windows/blob/main/PowerShellTricks.md#tls-12

The whole world uses TLS, but not older versions of PowerShell by default 😢

Reply:
Thanks, very useful and nice cheat sheet ;-)
In this case, however, it does not seem to be an issue with enabling TLS1.2 in old PWSH versions - just this once though :-)
The issue is with the certificate of the download.windowsupdate.com domain (hence Chrome, FF etc all also refusing/warning with a SSL_ERROR_BAD_CERT_DOMAIN: "Unable to communicate securely with peer: requested domain name does not match the server’s certificate.").

Doublechecked again if maybe I got a wrong link, but the download link in the pull request is exactly the once catalog.update.microsoft.com redirects you to (http only, no https).

As far as I can see looks like we'd have to accept http(no S), if we want the SP1 for Windows7.

Again, additional thoughts/suggestions more than welcome!

Reply:
OK, thanks for trying it out. 
Well, then we just keep it at http.

Reply:
Thanks too for your feedback.